src/App.js

```diff
@@ -1,18 +1,16 @@
-import React from "react"
-import './App.css';
-import Input from "./Input";
-import TextArea from "./TextArea";
+import React from "react";
 import Form from "./Form";
+import "./App.css";
 
 function App() {
-
   return (
-    <div className="App">
+    <div
```

Comment:
put this into a styled component... 
even if it's only two small styles, better to created a small container

```
const AppContainer = styled.div`
 background-color: blue,
 background-size: cover
`;
```
Avoid using inline styling unless absolutely necessary.